tests/test-coreos.sh

```diff
@@ -51,7 +51,7 @@ fetch_latest_rhcos_image() {
   if [[ $(echo "${shasum} ${image_path}/${latest_image_gz}" | sha256sum --check) != "${image_path}/${latest_image_gz}: OK" ]]; then
     mkdir -p ${image_path}
     rm -rf ${image_path}/*
-    wget -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
+    wget -q -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
```

Comment:
This turns off the output completely, but I suppose it would be still interesting to see some of it. What about `-nv`?

Reply:
```suggestion
    wget -nv -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
```

Reply:
totally, Done